spec/features/merchant/fulfill_order_shipped_spec.rb

```diff
@@ -0,0 +1,19 @@
+# require 'rails_helper'
```

Comment:
Should this test be uncommented? or still in progress?

Reply:
Commented. I skipped that after starting it because functionality to fulfill orders has not been completed.